mempool/reactor.go

```diff
@@ -112,6 +112,10 @@ func (memR *MempoolReactor) broadcastTxRoutine(peer p2p.Peer) {
 
 	var next *clist.CElement
 	for {
+		// In case of both next.NextWaitChan() and peer.Quit() are variable at the same time
```

Comment:
Thanks so much! Great find. Just the comment here is a bit confusing. Maybe:
```suggestion
		// Prevent infinite loop in case next != nil
```